From 0dd06574a286c705dd1d0798de56cf183d698b83 Mon Sep 17 00:00:00 2001
From: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 12 Jun 2013 13:24:19 +0300
Subject: [PORT FROM MAIN] Revert "Turn auto attach on/off when we attach/detach."

BZ: 102487

[DO NOT UPSTREAM]

This reverts commit f08c4c89622ddcbc4ed9315fe6b399941dad4939.

With the commit f08c4c89622ddcbc4ed9315fe6b399941dad4939,
framework keeps retrying the PDP context activation
even when the device is not registered to network.
Upon reregistering to network, framework will get
the registration status. So, no need to retry PDP context
activation when the device is out of service.

Change-Id: Ic02de51e5910ff06e1eacd22b4afab36ea023dcb
Orig-Change-Id: I6668c5c6c551401d0b0b4f2958c7dadca701afea
Signed-off-by: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 .../telephony/dataconnection/DcTracker.java        |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/java/com/android/internal/telephony/dataconnection/DcTracker.java b/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
index 8bc6479..5d598df 100644
--- a/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
+++ b/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
@@ -535,7 +535,6 @@ public final class DcTracker extends DcTrackerBase {
             // update APN availability so that APN can be enabled.
             notifyOffApnsOfAvailability(Phone.REASON_DATA_ATTACHED);
         }
-        mAutoAttachOnCreation = true;
         setupDataOnConnectableApns(Phone.REASON_DATA_ATTACHED);
     }
 
-- 
1.7.4.1

